.travis.yml

```diff
@@ -170,7 +170,7 @@ matrix:
           packages:
           - clang-7
           - llvm-7-dev
-    - env: BUILD_TYPE="Debug" WITH_SANITIZE="memory" INTEGER_CLASS="boostmp"
+    - env: BUILD_TYPE="Debug" WITH_SANITIZE="memory" INTEGER_CLASS="boostmp" CXX=clang++-7 CC=clang-7
```

Comment:
`CXX` here gets overriden by the `compiler: clang` below.

Reply:
ok, but how come e.g. the run with `WITH_SANITIZE="address"` [does this](https://travis-ci.org/symengine/symengine/jobs/512764111#L2285)?

Reply:
It's set at https://github.com/symengine/symengine/blob/master/bin/install_travis.sh#L106

Reply:
thanks!